File: src/codes/Users_Manual/controller_1.py
~# mypy: ignore-errors
~from datetime import timedelta
~from pyautd3 import Controller, AUTD3, Static, Null
~from pyautd3.link.audit import Audit
~autd: Controller[Audit] = Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Audit.builder())
~m = Static()
~g = Null()
autd.send((m, g).with_timeout(timedelta(milliseconds=20)))

File: src/codes/Users_Manual/controller_1.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder([AUTD3::new(Vector3::zeros())]).open(autd3::link::Nop::builder()).await?;
# let m = Static::new();
# let g = Null::new();
autd.send((m, g).with_timeout(std::time::Duration::from_millis(20))).await?;
# Ok(())
# }